app/views/sessions/new.html.erb

```diff
@@ -0,0 +1,19 @@
+<% provide(:title, "Log in") %>
+<h1>Log in</h1>
+
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for(:session, url: login_path) do |f| %>
```

Comment:
you have configured simple_form ..but you are using the default form builder, why?

Reply:
I tried using simple_form but it wasn't working so I just suspended using it for now